app/jobs/create_derivatives_job.rb

```diff
@@ -4,14 +4,21 @@ class CreateDerivativesJob < ApplicationJob
 
   # @param file_set_id [string] stringified Valkyrie id
   def perform(file_set_id)
-    Valkyrie::Derivatives::DerivativeService.for(id: file_set_id).create_derivatives
+    derivative_service = find_derivative_service(file_set_id)
+    derivative_service.create_derivatives
     file_set = query_service.find_by(id: Valkyrie::ID.new(file_set_id))
     file_set.processing_status = "processed"
     metadata_adapter.persister.save(resource: file_set)
     messenger.derivatives_created(file_set)
     CheckFixityJob.perform_later(file_set_id)
-  rescue Valkyrie::Persistence::ObjectNotFoundError => error
-    Valkyrie.logger.warn "#{self.class}: #{error}: Failed to find the resource #{file_set_id}"
+  rescue Valkyrie::Persistence::ObjectNotFoundError => not_found_error
+    Valkyrie.logger.warn "#{self.class}: #{not_found_error}: Failed to find the resource #{file_set_id}"
+  rescue StandardError => error
```

Comment:
Given https://github.com/pulibrary/figgy/pull/3040#issuecomment-502749459, should this proposed change be removed?  When I remove this but keep https://github.com/pulibrary/figgy/pull/3040/files#diff-575692548a61b37094b8e1c882bca938R47 in place, the tests do fail.